jerry-core/ecma/operations/ecma-objects.h

```diff
@@ -79,7 +79,9 @@ ecma_value_t ecma_op_is_regexp (ecma_value_t arg);
 ecma_value_t ecma_op_species_constructor (ecma_object_t *this_value, ecma_builtin_id_t default_constructor_id);
 ecma_value_t ecma_op_invoke_by_symbol_id (ecma_value_t object, lit_magic_string_id_t magic_string_id,
                                           ecma_value_t *args_p, uint32_t args_len);
-#endif /* JERRY_ESNEXT */
+void ecma_op_object_set_weak (ecma_object_t *const object, ecma_object_t *const target_object);
+void ecma_op_object_unref_weak (ecma_object_t *object_p, ecma_value_t ref_holder);
+#endif /* (JERRY_ESNEXT) */
```

Comment:
JERRY_BUILTIN_WEAKREF

Reply:
same as above

Reply:
There something weird about this guarding. Please check it again.